### PR TITLE
Setup environment variables action

### PR DESCRIPTION
**Pull request by vigneshmsft**
Action to set environment variables from a yaml file.

Co-authored-by: Vignesh.N <vigneshmsft@users.noreply.github.com>
Co-authored-by: Steve Fawcett <sfawcett123@users.noreply.github.com>

### CODE REVIEW

**Review by sfawcett123 — APPROVED** [review]
LGTM

File: .github/env.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TEAM_NAME: Teacher Services Infra
```
Comment (saliceti) [inline .github/env.yml:1]: This is specific to workflows, so it may make sense to move it under .github/workflows ?

Comment (sfawcett123) [reply .github/env.yml:1]: I wasn't comfortable with that, as the github system seems to consider yaml files in workflows as possible workflow files. so I took my lead from labeller.yml which has the workflow file where you would expect it, but the controlling yaml file in .github

Comment (vigneshmsft) [reply .github/env.yml:1]: yes, github could try and treat yml files inside workflows/ as a workflow definition file

**Pull request merged**